src/App.tsx

```diff
@@ -1,27 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+enum FUNCS {
+  GET_ALL = 'getAll',
+  GET_FIRST_5 = 'getFirstFive',
+  GET_RED = 'getRed',
+}
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [functionGetData, setFunctionGetData] = useState(FUNCS.GET_ALL);
```

Comment:
Do you really need this state for function data? You can remove it and refactor the code